src/unpackertool/plugins/upx/elf/elf_upx_stub.h

```diff
@@ -11,7 +11,9 @@
 
 #include "unpackertool/plugins/upx/upx_stub.h"
 #include "retdec/unpacker/decompression/nrv/bit_parsers.h"
-#include "retdec/unpacker/dynamic_buffer.h"
+#include "retdec/utils/dynamic_buffer.h"
+
+using namespace retdec::utils;
```

Comment:
Same as the comment I made earlier. Never put `using namespace` in header files.